src/fsharp/FSharp.Build-proto/FSharp.Build-proto.fsproj

```diff
@@ -1,57 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@dsyme, yes you are correct the only real difference in the build files is the path to proto.  On coreclr proto builds to fsc.dll because we can then use a host named fsc.exe to load it.
I will add a build switch to build that builds a desktop version of fsc.exe for the proto, just to simplify debugging.
Because I am ornery the default is going to be coreclr, but we can change that if it is too painful for the community.